prog/postgres/postgres_resource_nexus.rb

```diff
@@ -187,6 +187,7 @@ def before_run
 
     strand.children.each { _1.destroy }
     unless servers.empty?
+      servers.first.vm.private_subnets.each(&:incr_destroy)
```

Comment:
is it enough in all circumstances to delete private subnets of the first server?

Reply:
Yes, because they all share same private subnet. Also the first server is not special here, for all purposes we could use `servers.sample.vm.private_subnets.each(&:incr_destroy)`